subgraphs/goldfinch/src/mappings/senior_pool.ts

```diff
@@ -1,6 +1,6 @@
-import {Address} from "@graphprotocol/graph-ts"
+import { Address, log } from "@graphprotocol/graph-ts";
```

Comment:
A lot of the things to update in this contract are the same things I have said in others